client/src/views/SignUpPage.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="flex flex-col items-center justify-center font-custom">
+    <TodoLogin button-name="Log in" path="/login" />
+    <div
+      class="relative mt-2 flex w-72 flex-col items-center justify-center bg-white sm:mt-10 sm:w-screen sm:flex-row"
+    >
+      <div
+        class="w-64 rounded-xl border-2 bg-neutral-200 bg-opacity-60 p-7 shadow-lg sm:mb-10 sm:mr-60 sm:w-max sm:p-14"
+      >
+        <form name="login-form" class="sm:w-64">
+          <div class="mb-3 flex flex-col justify-between">
+            <label for="first-name" class="text-xs font-semibold sm:text-sm"
+              >First name</label
+            >
+            <input
+              type="text"
+              id="first-name"
+              class="focus:ring-primary-600 focus:border-primary-600 block w-full rounded-lg border border-gray-300 bg-gray-50 p-2 text-xs text-gray-900 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500 sm:p-2.5 sm:text-sm"
```

Comment:
why don't you move this to a custom input field, so you don't need to write this long class down for each input?